README.md

```diff
@@ -18,7 +18,7 @@ are carried out collectively by the network. Dash Core is the name of open
 source software which enables the use of this currency.
 
 For more information, as well as an immediately useable, binary version of
-the Dash Core software, see https://www.dashpay.io/downloads.
+the Dash Core software, see https://www.dash.org/downloads.
```

Comment:
This patch is incomplete - doesn't fix line number 4.
And it's already fixed in 12.1 btw.